mantidproject/mantid
# Tidy up pearl system tests

**Description of work.**

Tidy up two issues with the PEARL system tests found while working on #30163:

- the file Testing\Data\SystemTest\ISIS_Powder\input\calibration\pearl\17_1\pearl_offset_16_4.cal was being overwritten by the CreateCalTest with a different file. The other tests that use this file (eg FocusXXXTest and CreateVanadiumXXXTest) were set up to pass based on the overwritten .cal file rather than the one in GitHub. This was confusing and meant that the FocusXXXTest and CreateVanadiumXXXTest failed if run on their own
- the clean up of working files wasn't working properly with the summed_empty.nxs being left behind

**To test:**

No functional change so just code review and check Jenkins tests pass

Fixes #30924. 

*This does not require release notes* because it has no functional impact to users



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
